lib/section_reslugger.rb

```diff
@@ -58,8 +58,11 @@ def validate_new_section_in_database
   end
 
   def validate_new_section_in_content_store
-    section = section_in_content_store(full_new_section_slug)
-    raise Error.new("Manual Section already exists in content store") if section
+    begin
+      section = section_in_content_store(full_new_section_slug)
+      raise Error.new("Manual Section already exists in content store") if section
+    rescue GdsApi::ContentStore::ItemNotFound # rubocop:disable Lint/HandleExceptions
+    end
```

Comment:
It seems pretty nasty to use exception handling in this way. Is there no way to ask the Content Store whether the section exists and obtain a return value rather than an exception?